.gitignore

```diff
@@ -133,3 +133,4 @@ dmypy.json
 # IDE Settings
 .idea/
 .vscode/
+test.py
```

Comment:
It may be good to have test scripts in here too

Reply:
Yes this was just a script i was using to test with manually, we can add proper tests scripts in later though